requests/models.py

```diff
@@ -672,6 +672,7 @@ def __iter__(self):
 
     @property
     def ok(self):
+        """Returns true if :attr:`status_code` is 'OK'."""
```

Comment:
=D I already wrote up a PR for that. I'm also fixing up `__bool__` and `__nonzero__` at @Lukasa's request. Thanks for spurring all of this. =) :cake: